src/Rector/Contrib/PHPUnit/SpecificMethodCountRector.php

```diff
@@ -60,15 +74,25 @@ public function isCandidate(Node $node): bool
             return false;
         }
 
-        return $secondArgumentValue->name->toString() === 'count';
+        $coutableMethod = $secondArgumentValue->name->toString();
+
+        return $coutableMethod === 'count' || $coutableMethod === 'sizeof';
```

Comment:
For 2+ items to check one value agains, `in_array()` is more useful and easier to add more method names